website/docs/guides/features-block.html.markdown

```diff
@@ -150,6 +156,12 @@ The `log_analytics_workspace` block supports the following:
 
 ---
 
+The `managed_disk` block supports the following:
+
+* `no_downtime_resize` (Optional) Specifies if no-downtime resizes are enabled for the managed disk resources. Defaults to `true`.
```

Comment:
maybe use the same name as azure does for this feature enable_live_resize or something like that?

Reply:
Yeah agreed - digging around this it appears that `Expand Without Downtime` is the final name for this feature? I'd argue that `live_resize_data_disks_where_possible` (or something) would be clearer where possible, but since this is only for expanding (and not shrinking, which Azure doesn't support regardless of if the associated VM is online/offline) - I think it's probably worth making this `expand_without_downtime` for now?